[2023-12-08T23:28:38.291+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_processing_and_cleaning.clean_data_task manual__2023-12-08T23:28:36.763279+00:00 [queued]>
[2023-12-08T23:28:38.300+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_processing_and_cleaning.clean_data_task manual__2023-12-08T23:28:36.763279+00:00 [queued]>
[2023-12-08T23:28:38.300+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-08T23:28:38.313+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): clean_data_task> on 2023-12-08 23:28:36.763279+00:00
[2023-12-08T23:28:38.319+0000] {standard_task_runner.py:57} INFO - Started process 1268 to run task
[2023-12-08T23:28:38.321+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'data_processing_and_cleaning', 'clean_data_task', 'manual__2023-12-08T23:28:36.763279+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/data_processing_cleaning.py', '--cfg-path', '/tmp/tmpcbhhlcy3']
[2023-12-08T23:28:38.323+0000] {standard_task_runner.py:85} INFO - Job 23: Subtask clean_data_task
[2023-12-08T23:28:38.336+0000] {logging_mixin.py:154} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-12-08T23:28:38.370+0000] {task_command.py:416} INFO - Running <TaskInstance: data_processing_and_cleaning.clean_data_task manual__2023-12-08T23:28:36.763279+00:00 [running]> on host 7eda3474f8ff
[2023-12-08T23:28:38.469+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_processing_and_cleaning' AIRFLOW_CTX_TASK_ID='clean_data_task' AIRFLOW_CTX_EXECUTION_DATE='2023-12-08T23:28:36.763279+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-08T23:28:36.763279+00:00'
[2023-12-08T23:29:10.218+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-12-08T23:29:10.227+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=data_processing_and_cleaning, task_id=clean_data_task, execution_date=20231208T232836, start_date=20231208T232838, end_date=20231208T232910
[2023-12-08T23:29:10.299+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-08T23:29:10.313+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
